LabelGenerator.py

```diff
@@ -602,6 +607,19 @@ def render_outlines(c: Canvas, layout: PaperConfig) -> None:
                 c.setLineWidth(0)
                 c.roundRect(rect.left, rect.bottom, rect.width, rect.height, rect.corner)
 
+def generate_decade(
+        series: List[float], # the base series of resistor values
+        power: int           # the power of 10 to scale the series by
+) -> List[float]:
+    scalar = 10 ** power
+    return [scalar * x for x in series]
+
+def generate_decades(
```

Comment:
Please add documentary to this function, with all parameters explained und what it generates. Also mention all the possible values for the `series` parameter.

Also, I think the name is misleading. What it actually does is to generate commonly used values, so maybe name it "generate_common_values" or similar?

Reply:
Documentation added, with examples